project.clj

```diff
@@ -43,7 +43,7 @@
                  [puppetlabs/kitchensink ~ks-version]
                  [puppetlabs/trapperkeeper ~tk-version]
                  [puppetlabs/trapperkeeper-scheduler "0.0.1"]
-                 [puppetlabs/ring-middleware "1.0.0"]
+                 [puppetlabs/ring-middleware "1.0.1" :exclusions  [org.clojure/clojure org.slf4j/slf4j-api]]
```

Comment:
Rather than doing `exclusions` like this we've been pinning the version in a "dependency conflict resolution" list - e.g. https://github.com/puppetlabs/trapperkeeper-status/pull/65/files#diff-0fff143854a4f5c0469a3819b978a483R27.

Probably this just means bumping the `slf4j-api` version in the conflict list?